src/MongoDB/Manager.c

```diff
@@ -349,6 +349,26 @@ static PHP_METHOD(Manager, __construct)
 	}
 } /* }}} */
 
+/* {{{ proto MongoDB\Driver\ClientEncryption MongoDB\Driver\Manager::clientEncryption(array $options)
+   Return a ClientEncryption instance */
+static PHP_METHOD(Manager, clientEncryption)
```

Comment:
Other drivers allow creating a `ClientEncryption` object through the constructor. This isn't an option for us, as we'd need to unpack the `MongoDB\Driver\Manager` instance from a `keyVaultClient` option. Since this requires logic in PHPLIB, we opted to expose the class through the Manager and Client classes.

I'm perfectly fine prefixing the method, and considering that it always creates a new `ClientEncryption` instance, I'd go for `createClientEncryption`.